libs/domains/services/feature/src/lib/general-setting/general-setting.tsx

```diff
@@ -38,6 +40,21 @@ export function GeneralSetting({ label = 'Name' }: GeneralSettingProps) {
           />
         )}
       />
+      {type && (
```

Comment:
I think you can keep the original component and introduce in each `useForm` data where `GeneralSettings` is been used a new `template_type` property.
You can then set `defaultValues` of each forms depending on the service type, creating an alias on `schedule.lifecycle_type` if the service is a job.
We can then supports any service template.

Reply:
https://github.com/Qovery/console/pull/1465/commits/03007b3e64976bef9151df12b8c4951951f95b24
https://github.com/Qovery/console/pull/1465/commits/a2a138c0cf82fd0c42655aeee0078f17c329344c